index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex-container header-items">
+    <a href="#" id="header-welcome-link">Welcome</a>
+    <img src="images/union.png" alt="nav button">
+  </header>
+  <section class="flex-container bg-img">
+    <div>
+      <p class="headline-paragraphs" id="p1">Hey there. I’m Mateo<br><span id="p3">I'm a software developer</span></p> 
+      <p class="headline-paragraphs" id="p2">I can help you build a product, feature or website Look through 
+        some of my work and experience! If you like what you see and have 
+        a project you need coded, don’t hestiate to contact me. </p> 
+    </div>
+    <nav class="flex-container nav">
+      <ul>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/twitter.svg" alt="Twitter Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/linkedin.svg" alt="Linked Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/medium.svg" alt="Medium Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/github.svg" alt="Github Link">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="images/socialLinks/angel.svg" alt="Angel Link">
+          </a>
+        </li>
+      </ul>
```

Comment:
Center the icons as now the space is not equal on both sides 

#### Original design
![right1](https://user-images.githubusercontent.com/58376020/124958866-e9ba3c00-e01a-11eb-8606-0ff25458f95e.png)

#### Your project
![wrong1](https://user-images.githubusercontent.com/58376020/124958896-f474d100-e01a-11eb-956d-0cb5eda712e5.png)

Reply:
Resolving this issue was a little bit tricky :exploding_head: